### ISSUE TITLE
Request accessibility/508 review for PRODUCT_NAME

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-vfs-teams#35]
## Team Name

Booz Allen External Team

## Product Description

Working on: STEM Scholarship Form MVP

Context:  Form 22-1995 has been updated to allow veterans to apply for the STEM Scholarship.

## Staging URL and Credentials

URL: https://staging.va.gov/education/apply-for-education-benefits/application/1995/applicant/information
Login: No login required 

## Known Issues
Contact Desiree Turner (@desireeturner) with any issues found. Please add labels "bah" and "bah-ct-116" to any issues found.
The following issues are in the process of being resolved by our team: 
- https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19374
- https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19373
- https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19379

## Helpful Links

- [508 Accessibility Issue
  Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=508-issue.md&title=)

**Comment by KevinMHoffmanUSDS** [comment]
**6 errors (1 type of error) on landing page**
https://staging.va.gov/gi-bill-comparison-tool/
Fix requested: on 
autocomplete-valid: autocomplete attribute must be used correctly - radio button issue
WCAG 1.3.5
Details of the requirement: https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html

Affected buttons ALL radio-buttons

E.g.

<input autocomplete="false" id="radio-buttons-3-0" name="category" type="radio" value="school" checked="">

Fix all of the following: the autocomplete attribute is incorrectly formatted the autocomplete value is inappropriate for this type of input

**No errors on search results page**

**Two errors (1 type of error) on profile page**
https://staging.va.gov/gi-bill-comparison-tool/profile/2V000232

Details of the requirement: 
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships
https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

2 failures were detected. When the Automated Checks toggle is on, failure instances selected in this page are highlighted in the target page.

label: Form elements must have labels  WCAG 1.3.1, WCAG 3.3.2

Example: 
input[name="vetTecTuitionFees"]
<input type="text" name="vetTecTuitionFees" value="$0">
Fix any of the following: aria-label attribute does not exist or is empty aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty Form element does not have an implicit (wrapped) <label> Form element does not have an explicit <label> Element has no title attribute or the title attribute is empty

**Comment by KevinMHoffmanUSDS** [comment]
@desireeturner You can see the results of the 508 review of the three pages (landing, results, profile) above. There are a total of 8 errors on the first and last page in the sequence. We'll still need a review from @1Copenut as soon as it's possible, but these are a few errors you could probably get ahead of before that review.

**Comment by 1Copenut** [comment]
@desireeturner I was able to get through the audit today with screen readers. I'll compare notes with the comment above from @KevinMHoffmanUSDS and write issue tickets tonight and tomorrow AM.
